components/search_results/search_results.tsx

```diff
@@ -285,16 +286,14 @@ const SearchResults: React.FC<Props> = (props: Props): JSX.Element => {
         contentItems = sortedResults.map((item: Post|FileSearchResultItemType, index: number) => {
             if (searchType === DataSearchTypes.MESSAGES_SEARCH_TYPE && !props.isChannelFiles) {
                 return (
-                    <SearchResultsItem
+                    <PostComponent
```

Comment:
This will show reactions as well for search items that we didn't show up until now, right?

If that's the case, that reactions are displayed whenever a post is displayed, then we can delete `showDivider` prop in `PostAcknowledgements`

Reply:
@koox00 I am not sure about the desired behaviour here. Do you mean post acknowledgements or emoji reactions? @jgilliam17 should we display either of those in search results?

Reply:
@nevyangelova We display acknowledgments in search, pinned, saved and mentions. We do not display post reactions.